docs/Other SDL Features/Block SdlRouterService by vehicle type/index.md

```diff
@@ -0,0 +1,58 @@
+# Block SdlRouterService by vehicle type
+The Android library can prevent apps from starting their SdlRouterService based on if they support the vehicle type if the connection is over Bluetooth.
+
+!!! NOTE
+If older apps are installed that do not utilize `SdlDeviceListener`, on the first connection to the vehicle an OEM app may host the `SdlRouterService` but on subsequent connections it may pass it off to another app.
```

Comment:
Can we simplify this sentence? I mean developers may not know what `SdlDeviceListener` is because it is something used only internally in the library. Maybe we can just specify a library version instead?